### PR TITLE
MEN-7277 - show SSO info depending on configured SSO type

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

**Review by merlin-northern — COMMENTED** [review]
there is a slight problem when I choose OIDC as SSO method (see also the recording on priv):
in the dev mode:
```json
helpers.js:534 Uncaught TypeError: _constants_organizationConstants_js__WEBPACK_IMPORTED_MODULE_6__.SSO_TYPES.find is not a function
    at getSsoByType (helpers.js:534:47)
    at organization.js:162:40
    at handleChange (InputBase.js:364:1)
    at SelectInput.js:291:1
    at HTMLUnknownElement.callCallback (react-dom.development.js:4164:1)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:4213:1)
    at invokeGuardedCallback (react-dom.development.js:4277:1)
    at invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:4291:1)
    at executeDispatch (react-dom.development.js:9041:1)
    at processDispatchQueueItemsInOrder (react-dom.development.js:9073:1)
```

and in regular prod build:
```json
helpers.js:534 Uncaught TypeError: u.c0.find is not a function
    at ne (helpers.js:534:47)
    at organization.js:162:28
    at onChange (InputBase.js:364:7)
    at SelectInput.js:291:9
    at Object.Re (react-dom.production.min.js:54:317)
    at ze (react-dom.production.min.js:54:471)
    at react-dom.production.min.js:55:35
    at Nn (react-dom.production.min.js:105:68)
    at Bn (react-dom.production.min.js:106:380)
    at react-dom.production.min.js:117:104
```

**Comment by mzedel** [comment]
Thanks for taking a look @merlin-northern! Based on this error I looked again into the API specs for the SSO config retrieval [here](https://docs.mender.io/api/#management-api-user-administration-and-authentication-get-saml-or-openid-connect-metadata-for-the-tenant) and I think for this task to provide a correct set of links the backend needs to return the type of config along with the `id` and `issuer`...

**Comment by merlin-northern** [comment]
> Thanks for taking a look @merlin-northern! Based on this error I looked again into the API specs for the SSO config retrieval [here](https://docs.mender.io/api/#management-api-user-administration-and-authentication-get-saml-or-openid-connect-metadata-for-the-tenant) and I think for this task to provide a correct set of links the backend needs to return the type of config along with the `id` and `issuer`...

I can fix that.

File: src/js/actions/userActions.js [hunk 1]
```diff
@@ -81,8 +81,9 @@ export const loginUser = (userData, stayLoggedIn) => dispatch =>
...
       // user should be redirected to the start sso url to finish login process.
       if (contentType.includes(APPLICATION_JSON_CONTENT_TYPE)) {
-        const { id } = response;
-        const ssoLoginUrl = getSsoStartUrlById(id);
+        const { id, kind } = response;
+        const type = kind.split('/')[1];
+        const ssoLoginUrl = SSO_TYPES[type].getStartUrl(id);
         window.location.replace(ssoLoginUrl);
         return;
       }
```
Comment (merlin-northern) [inline src/js/actions/userActions.js:89]: I havent tested this flow, lets do it on staging.

**Review by merlin-northern — APPROVED** [review]
looks good to me. also tested with the SAML and OIDC metadata upload and start of the login process.

**Pull request merged**